tests/fsharp/core/unicode/test.fsx

```diff
@@ -24,6 +24,8 @@ let input_byte (x : System.IO.FileStream) =
     let b = x.ReadByte() 
     if b = -1 then raise (System.IO.EndOfStreamException()) else b
 
+let bslFilePath = Path.Combine(__SOURCE_DIRECTORY__,"out.bsl")
```

Comment:
This test had the hard-coded assumption about current directory.
Instead replacing by using __SOURCE_DIRECTORY__